asclepias_broker/config.py

```diff
@@ -0,0 +1,253 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 CERN.
+#
+# Asclepias Broker is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Default configuration for Asclepias Broker."""
+
+from __future__ import absolute_import, print_function
+
+import os
+from datetime import timedelta
+
+from invenio_app.config import APP_DEFAULT_SECURE_HEADERS
+from invenio_records_rest.facets import terms_filter
+from invenio_records_rest.utils import deny_all
+from invenio_search.api import RecordsSearch
+
+from asclepias_broker.search import enum_term_filter, nested_range_filter, \
+    nested_terms_filter
+
+
+def _(x):
+    """Identity function used to trigger string extraction."""
+    return x
+
+
+# Rate limiting
+# =============
+
+RATELIMIT_STORAGE_URL = 'redis://localhost:6379/3'
+
+# I18N
+# ====
+#: Default language
+BABEL_DEFAULT_LANGUAGE = 'en'
+#: Default time zone
+BABEL_DEFAULT_TIMEZONE = 'Europe/Zurich'
```

Comment:
Is there a way to make this more generic? I think there are ways to figure out the current time zone in Python?